website.Rmd

```diff
@@ -2,17 +2,297 @@
+
+-   The convention is to orient the logo with a vertex at the top and bottom, with flat vertical sides.
+
+-   If you think you might print stickers, make sure to comply with the *de facto* standard for sticker size.
+    [hexb.in](http://hexb.in/sticker.html) is a reliable source for the dimensions and also provides a list of potential vendors for printed stickers.
+
+    ```{r}
+    #| label: fig-hex-sticker-spec
+    #| echo: false
+    #| out.width: 35%
+    #| fig.cap: Standard dimensions of a hex sticker.
+    # https://github.com/terinjokes/StickersStandard/blob/master/assets/hex-image.png
+    knitr::include_graphics("diagrams/hex-image.png")
+    ```
+
+-   The [hexSticker package](https://cran.r-project.org/package=hexSticker) helps you make your logo from within the comfort of R.
+
+Once you have your logo, the `usethis::use_logo()` function places an appropriately scaled copy of the image file at `man/figures/logo.png` and also provides a copy-paste-able markdown snippet to include your logo in your `README`.
+pkgdown will also discover a logo placed in the standard location and incorporate it into your site.
+
+## Reference index
+
+pkgdown creates a function reference in `reference/` that includes one page for each `.Rd` help topic in `man/`.
+This is one of the first pages you should admire in your new site.
+As you look around, there are a few things to contemplate, which we review below.
+
+pkgdown executes all your examples (@sec-man-examples) and inserts the rendered results.
+We find this is a fantastic improvement over just showing the source code.
+This view of your examples can be eye-opening and often you'll notice things you want to add, omit, or change.
+If you're not satisfied with how your examples appear, this is a good time to review techniques for including code that is expected to error (@sec-man-examples-errors) or that can only be executed under certain conditions (@sec-man-examples-dependencies-conditional-execution).
+
+These help topics will be linked to from many locations within and, potentially, beyond your pkgdown site.
+This is what we are talking about in @sec-man-key-md-features when we recommend putting functions inside square brackets when mentioning them in a roxygen comment:
+
+```{r}
+#' I am a big fan of [thisfunction()] in my package. I
+#' also have something to say about [otherpkg::otherfunction()]
+#' in somebody else's package.
+```
+
+On pkgdown sites, those square-bracketed functions become hyperlinks to the relevant pages in your pkgdown site.
+This is automatic within your package.
+But links from *other* people's packages (and websites, etc.) require two things:
+
+-   Your DESCRIPTION file must include URLs for your pkgdown site and GitHub repo:
+
+    ``` yaml
+    URL: https://dplyr.tidyverse.org, https://github.com/tidyverse/dplyr
+    ```
+
+-   Your `_pkgdown.yml` file must include the URL for your site:
+
+    ``` yaml
+    url: https://dplyr.tidyverse.org
+    ```
```

Comment:
@Hadley I'm looking for some confirmation on this. In particular, I feel like I've read somewhere that the pkgdown site's URL must be in DESCRIPTION but then it neglects to mention that GitHub must be there too (obviously I should try to remember where I saw that).

Yet it seems like the GitHub URL would also be a requirement, yeah? In order to find `_pkgdown.yml`?

Reply:
I must be thinking of the language from pkgdown's vignette on linking:

https://pkgdown.r-lib.org/articles/linking.html#across-packages

> In order for a pkgdown site to be findable, it needs to be listed in two places:
> 
> * In the URL field in the DESCRIPTION, as in dplyr:
> 
>     URL: https://dplyr.tidyverse.org, https://github.com/tidyverse/dplyr
> * In the url field in `_pkgdown.yml`, as in dplyr
>
>    url: https://dplyr.tidyverse.org
> ...
> Now, when you build a pkgdown site for a package that links to the dplyr documentation (e.g., `dplyr::mutate()`), pkgdown looks first in dplyr’s DESCRIPTION to find its website, then it looks for `pkgdown.yml`, and uses the metadata to generate the correct links.

Now that I've done a close reading, I conclude that, while a good idea, the GitHub URL has nothing to do with pkgdown's auto-linking. Instead this is the critical file, while comes from the pkgdown site itself:

> pkgdown generate a public facing `pkgdown.yml` file that provides metadata about the site